UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Home/Error.cshtml

```diff
@@ -0,0 +1,17 @@
+﻿<div class="container-fluid">
+    <div class="row">
+        <div class="offset-md-2 col-lg-6 col-md-8 custom-error">
+            <h1 id="headingLevelOne" class="mt-0">Sorry, there is a problem with the service</h1>
+            <p class="intro-text">Please try again later</p>
+            <p class="paragraph">
+                If you need to talk to someone, contact the customer service team at <a href="mailto:customerservice@ukho.gov.uk?subject=CorrelationId: @Convert.ToString(ViewData["CurrentCorrelationId"])">customerservice@ukho.gov.uk</a>.
+                <br />
+                Use CorrelationId: <b>@Convert.ToString(ViewData["CurrentCorrelationId"])</b> for your reference.
```

Comment:
Suggested change to reference message:
```suggestion
                Please use <span class="font-monospace fw-bold">@Convert.ToString(ViewData["CurrentCorrelationId"])</span> as your reference.
```

Reply:
![image](https://user-images.githubusercontent.com/91606761/173564803-3379f536-195f-4db2-b743-c4c3ebce6c9d.png)

Reply:
Added suggested reference message